.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve `php-github-api`
+
+---
+
+**Configuration (preferably required)**
+- **php-github-api version:** "...replace me..."
+- **PHP version:** "...replace me (Can be retrieved using command line `php -v | sed -n 1p`)..."
+- **Operating system:** " ...replace me (You REALLY should know it)... "
+
+**Describe the bug**
+
+> ... A clear and concise description of what the bug is, goes here ... 
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
```

Comment:
Screenshots of what? Is this really useful for 80% of issues?

Reply:
Ok, I'll remove it.